### DISCUSSION TITLE
Github Student developer pack Approved but didn't get the benefits mail

### DISCUSSION THREAD

**Discussion by akarshansonkar** [community/community#112398] [GitHub Education]
### Hi everyone,

can you please let me know how much time it's gonna take to receive the mail regarding this , it is showing approved but didmn't receive the mail yet.

**Comment by SebastienXia** [comment]
Hi, @akarshansonkar 

You will receive an email which when you'll get access to the benefits of the Pack don't worry, the process can take some times like 6 - 7 days after you get your approved demand...
